src/scripts/utils/playerUtils.js

```diff
@@ -368,7 +368,16 @@ playerUtils.prepareForAds = function (player) {
       var origSrc = player.src;
       player.src = function (source) {
         if (source && !(source instanceof parent.Object)) {
-          source = parent.Object.assign(new parent.Object(), source);
+          if (utilities.isIE11()) {
+             var temp = new parent.Object();
+             for (var prop in source) {
```

Comment:
It would probably be prudent to add a `if (source.hasOwnProperty(prop)) {` check here, to be sure you don't pick up anything off the prototype. It's unlikely to be a problem, since these are just raw Objects, but it's still best practice.

Reply:
Did it.